frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/AccountInterventionsHandler.java

```diff
@@ -139,14 +133,21 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                             accountInterventionsInboundResponse.state().suspended());
             return generateApiGatewayProxyResponse(200, accountInterventionsResponse, true);
         } catch (UnsuccessfulAccountInterventionsResponseException e) {
+            LOG.info("UNSUCCESSFUL ACCOUNT INTERVETNIONS RESPONSE {}", e);
             return handleErrorForAIS(e);
         } catch (JsonException e) {
+            LOG.info("JSONexception");
             return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1001);
         }
     }
 
     private APIGatewayProxyResponseEvent handleErrorForAIS(
             UnsuccessfulAccountInterventionsResponseException e) {
+        LOG.info("AIS Exception should be occuring here");
+        LOG.info("AIS Exception should be occuring here");
```

Comment:
duplicate line?

Reply:
This commit has been removed now, got pushed to this branch by mistake